connectors/src/connectors/notion/index.ts

```diff
@@ -142,10 +125,7 @@ export async function fullResyncNotionConnector(connectorId: string) {
   }
 
   try {
-    await stopNotionConnector({
-      workspaceId: connector.workspaceId,
-      dataSourceName: connector.dataSourceName,
-    });
+    await stopNotionConnector(connector.id.toString());
```

Comment:
Wondering if we shouldn't make the interface take a number instead of  toStringing everytime

Reply:
Let's type it as an alias of a number?

Reply:
I decided to implement that in the follow up PR because it is too much work and would make that PR too big.